build/plugins/attributes.html

```diff
@@ -0,0 +1,132 @@
+<!-- attributes plugin -->
+
+<script>
+    (function() {
+        // /////////////////////////
+        // DESCRIPTION
+        // /////////////////////////
+
+        // This Manubot plugin allows arbitrary HTML attributes to be attached
+        // to (almost) any element. Place an HTML comment inside or next to the
+        // desired element in the format <!-- attribute="value" -->
```

Comment:
Can the comment elaborate on the behavior of "next to the desired" element? Also should we describe the colspan special case behavior?

Reply:
I don't know how to elaborate on that in a helpful way, because it varies for every element.

If it's a link, you put it in between the square brackets. If it's a header, put it right after the text on the same line. If it's a paragraph, put it at the end of any sentence.

It depends on how Pandoc passes the comment through and what element it ends up attaching the comment to, and that decision is somewhat arbitrary, dependent on pandoc, not documented by pandoc (I assume), and might also be subject to change in the future.

Reply:
Ah I see. The attributes apply to whatever element pandoc places them in. The "next to the desired element" confused me. Let's tweak the documentation to elaborate on this point.